brightener.h

```diff
@@ -6,7 +6,5 @@ class ImageBrightener {
 	std::shared_ptr<Image> m_inputImage;
 public:
 	ImageBrightener(std::shared_ptr<Image> inputImage);
-	bool ValidateImage();
 	int BrightenWholeImage();
```

Comment:
Nice removal of unwanted functions. 
But does `BrightenWholeImage` need to be in a class with a data member? Or could it just be a function that accepts the image to be processed?